node/v2/handler.js

```diff
@@ -289,6 +293,14 @@ TChannelV2Handler.prototype.handleCallResponse = function handleCallResponse(res
                 limit: '0x' + v2.CallResponse.MaxArg1Size.toString(16)
         }));
     }
+
+    var req = self.connection.ops.getOutReq(res.id);
+    self.connection.channel.inboundResponseSizeStat.increment(resFrame.size, {
+        'calling-service': !req ? null : req.headers.cn,
```

Comment:
@rf it used to be unknown. Then @kriskowal suggested the current name. :) I don't really have any preference on the default names. They shouldn't be showing up in almost any of these cases.